_scripts/ProcessLocalesPlugin.js

```diff
@@ -37,66 +38,88 @@ class ProcessLocalesPlugin {
 
       compilation.hooks.additionalAssets.tapPromise('process-locales-plugin', async (_assets) => {
 
-        // While running in the webpack dev server, this hook gets called for every incrememental build.
+        // While running in the webpack dev server, this hook gets called for every incremental build.
         // For incremental builds we can return the already processed versions, which saves time
         // and makes webpack treat them as cached
-        if (IS_DEV_SERVER && this.cache.length > 0) {
-          for (const { filename, source } of this.cache) {
-            compilation.emitAsset(filename, source, { minimized: true })
-          }
-        } else {
-          const promises = []
-
-          for (const { locale, data } of this.locales) {
-            promises.push(new Promise(async (resolve) => {
-              if (Object.prototype.hasOwnProperty.call(data, 'Locale Name')) {
-                delete data['Locale Name']
+        const promises = []
+        this.loadLocales(true)
```

Comment:
Currently what this does is calls loadLocales once when the plugin is created in the constructor (can't be removed as we need to be able to pass the locale names to the DefinePlugin) and then straight away again when this hook is called, when webpack goes to compile the files, so please add an `if` around this, so it is only called when the hook is called the second, third, fourth, etc times, not on the first time. (that will also prevent loadLocales getting called twice for release builds).

Can be a simple if else e.g.

```js
// in constructor
this.isIncrementalBuild = false

// here
if (this.isIncrementalBuild) {
  this.loadLocales(true)
} else {
  this.isIncrementalBuild = true)
}